pkg/crc/preflight/preflight_linux.go

```diff
@@ -218,14 +218,6 @@ func usesSystemdResolved(distro *linux.OsRelease) bool {
 	}
 }
 
-func distroID(osRelease *linux.OsRelease) linux.OsType {
-	if osRelease == nil {
```

Comment:
`distro()` can return `nil` if `linux.GetOsRelease()` fails, so it's possible to get a `nil` `*linux.OsRelease` here.

Reply:
outch, thanks!

Reply:
done